docs/updating_schemas.md

```diff
@@ -3,29 +3,31 @@ Making changes to schemas
-    * **audit** - Contains Python scripts that check JSON objects' metadata stored in the schema
-    * **schemas** - JSON schemas ([JSONSchema], [JSON-LD]) describing allowed types and values for all metadata objects
-    * **schemas/changelogs** - Schema change logs for documenting different versions of schemas
-    * **schemas/mixins.json** - Common schema property definitions for use with multiple schemas
-    * **tests** - Unit and integration tests
-    * **tests/data/inserts** - The sample data that comes up in a basic local instance.
-    * **tests/fixtures/schemas** - The sample Python objects to use with unit tests.  These Python function names are used as parameters in unit tests.
-    * **types** -  Business logic for dispatching URLs and producing the correct JSON
-    * **upgrade** - Python instructions for upgrading old objects to match the new schema
-    * **loadxl.py** - Python script that defines the schema objects to load
-    * **schema_format.py** - The format checker for accessions
-
-
------
+* **src/igvfd** Directory - contains all the Python code and JSON schemas for adding new types to the igvfd backend.
+    * **/audits** - Contains Python scripts that are run post-submission to check JSON objects' metadata stored in the schema
+    * **/schemas** - JSON schemas ([JSONSchema], [JSON-LD]) describing allowed types and values for all metadata objects
+      * **/schemas/changelogs** - Schema change logs for documenting the changes made in each version of a schema
+      * **/schemas/mixins.json** - Common schema property definitions for use with multiple schemas
+    * **/tests** - Unit and integration tests
+      * **/tests/data/inserts** - The sample data that comes up in a basic local instance.
+      * **/tests/fixtures/schemas** - The sample Python objects to use with unit tests.  These Python function names are used as parameters in unit tests.
+    * **/types** -  Logic for dispatching URLs and producing the correct JSON.  This is where calculated properties are specified.
+    * **/upgrade** - Python instructions for upgrading old objects to match the new schema
+    * **/loadxl.py** - Python script that defines the schema objects to load
+    * **/schema_format.py** - The format checker for accessions
+    * **/searches** - Directory that contains the configurations for search results
+      * **/searches/defaults.py** - This file contains a list that determines if a new schema is searched by default
+      * **/searches/configs** - Directory that has a file for each type that contains the list of fields returned in a search
 
 Adding a new schema
 ----------------
 
-1. To add a new schema navigate to the **schemas** directory and make a new JSON file named after the object. Populate the file with basic schema definition:
+1. Add a new JSON file to the **schemas** directory named after the object (i.e. antibody.json). Populate the file with basic schema definition:
 
 
             {
```

Comment:
Should we add an entry in this block for "dependentSchemas"?

Reply:
Likely, do you have thoughts on what we should add?

Reply:
Would it be OK to leave it blank, as we did with the "required" entry?
Otherwise, something like the below is probably the simplest form of dependency we have:
```
"duration": {
            "required": [
                "duration_units"
            ],
            "comment": "duration_units must be specified if duration is specified"
        },
```

Reply:
Resolved by adding step 4 below